python/tchannel/tornado/message_factory.py

```diff
@@ -56,6 +58,31 @@ def __init__(self, remote_host=None, remote_host_port=None):
         self.in_checksum = {}
         self.out_checksum = {}
 
+    def build_raw_error_message(self, protocol_error):
+        """build protocol level error message based on Error object"""
+        message = ErrorMessage(
+            code=protocol_error.code,
+            tracing=protocol_error.Tracing(
+                protocol_error.tracing.span_id,
+                protocol_error.tracing.parent_span_id,
+                protocol_error.tracing.trace_id,
+                protocol_error.tracing.traceflags),
+            description=protocol_error.description,
+        )
+
+        return message
+
+    def build_protocol_error(self, message, message_id=None):
```

Comment:
As mentioned before: I don't think these methods belong in this class. This class context is providing no value to the instantiation of these objects.